fmf/utils.py

```diff
@@ -955,6 +957,7 @@ def validate_data(data, schema, schema_store=None):
             jsonschema.exceptions.RefResolutionError,
             jsonschema.exceptions.UnknownType
             ) as error:
+        from fmf.utils import JsonSchemaError
```

Comment:
Unneeded? We are in `utils.py`

Reply:
You forgot this one in the recent rebase?

Reply:
Huh, I specifically checked for this. Must be going insane. Thanks mate.